.ci/run.sh

```diff
@@ -24,7 +24,7 @@ test_repo_dir="${GOPATH}/src/${test_repo}"
 
 cidir=$(dirname "$0")
 
-sudo -E PATH=$PATH bash -c  "make check"
+sudo -E PATH=$PATH CI=true bash -c "make check"
```

Comment:
Shouldn't that be `CI=$CI`? All the CI's set that variable so we just want to pass the value down to `make` and all it's children I think.

Reply:
Yes that makes sense.